osquery/tables/applications/posix/carbon_black.cpp

```diff
@@ -88,8 +90,8 @@ void getQueue(Row& r) {
   if (!listFilesInDirectory(kCbDir, files_list, true)) {
     return;
   }
-  unsigned int binary_queue_size = 0;
-  unsigned int event_queue_size = 0;
+  uintmax_t binary_queue_size = 0;
```

Comment:
I think a `std::size_t` works too, but best to go with what the API returns. Nice.